index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Trial Project</title>
+    <style>
+      * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+      }
+      .bt {
+        background-color: #04aa6d;
+        font-size: large;
+        width: 100%;
+        padding: 12px 20px;
+        font-weight: bold;
+        border: none;
+        color: aliceblue;
+        border-radius: 3px;
+      }
+      header{
+        width: 100%;
```

Comment:
I would suggest you write the CSS in a separate file. This way it will be easier to organize your project.